comp/core/workloadmeta/collectors/internal/kubeapiserver/kubeapiserver.go

```diff
@@ -47,14 +47,19 @@ func storeGenerators(cfg model.Reader) []storeGenerator {
 		generators = append(generators, newDeploymentStore)
 	}
 
+	addNamespaceStore := len(cfg.GetStringMapString("kubernetes_namespace_labels_as_tags")) > 0
```

Comment:
We should also check `kubernetes_namespace_annotations_as_tags`.

Reply:
Nice catch !